rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -461,6 +461,32 @@ public void release_verify_fee_above_fee_and_minimum_is_accepted() throws IOExce
         testPegoutMinimumWithFeeVerification(Coin.COIN, Coin.FIFTY_COINS, true);
     }
 
+    @Test
+    public void processReleasesInBatch_before_rskip_271() throws IOException {
+        when(activationMock.isActive(ConsensusRule.RSKIP271)).thenReturn(false);
+
+        bridgeSupport.releaseBtc(releaseTx);
+
+        Transaction rskTx = buildUpdateTx();
+        bridgeSupport.updateCollections(rskTx);
+
+        assertEquals(0, provider.getReleaseRequestQueue().getEntries().size());
+        assertEquals(1, provider.getReleaseTransactionSet().getEntries().size());
+    }
```

Comment:
I think we have the chance to test different scenarios here. 

- 0, 1, or multiple release requests made
- Pre RSKIP 271 I guess there should be one element in the release transaction set per pegout request
- Post RSKIP 271 there should be only one element in the release transaction, right?
- Create some pegout requests, call updateCollections. Create some more pegout requests, call updateCollections again. Are requests being batched? Maybe mock the provider to return different values for the nextPegoutHeight, that should let us cover some scenarios

Reply:
Most of These Test Cases Would Be Handled In The Update ReleaseTransactionBuilder Task
For Now, It Is Assumed There Are No Transactions In The Queue Because The BatchPegout Method Hasn't Been Implemented

Reply:
Ok, let's just keep them in mind